### PR TITLE
test:Add a test case for errorWrapper.ts

### PR DESCRIPTION
**Pull request by doz13189**
this PR is for issue #419 .

If the component has no classes, vueWrapper returns createWrapperError. createWrapperError error cannot be caught by expect(), so I caught it with try-catch.

I'm a beginner, so I'm sorry if I make a mistake.

### CODE REVIEW

File: tests/classes.spec.ts [hunk 1]
```diff
@@ -55,5 +55,22 @@ describe('classes', () => {
...
+        }
+      })
+
+      const wrapper = mount(Component)
+      try {
+        wrapper.find('.class-c').classes()
+      } catch (error) {
+        expect(error.message).toEqual(
+          'Cannot call classes on an empty DOMWrapper.'
+        )
```
Comment (cexbrayat) [inline tests/classes.spec.ts:72]: You can replace the try/catch here with `expect(() => wrapper.find...).toThrow(...)`
See https://jestjs.io/docs/en/expect.html#tothrowerror

File: tests/classes.spec.ts [hunk 2]
```diff
@@ -55,5 +55,22 @@ describe('classes', () => {
       expect(wrapper.classes('class-a')).toBe(true)
       expect(wrapper.classes('class-b')).toBe(false)
     })
+
+    it('returns error if the component does not have the class', () => {
```
Comment (cexbrayat) [inline tests/classes.spec.ts:59]: nit: `returns error if the component does not have the class` => `throws an error if called on an empty wrapper`

**Review by cexbrayat — CHANGES_REQUESTED** [review]
Looks good overall, thanks for the PR 👍

I left 2 comments for you to fix.

Can you also change the commit message to have a space and be in lowercase please? `test: add a test case for errorWrapper`

File: tests/classes.spec.ts (hunk 1 above)
Comment (doz13189) [reply tests/classes.spec.ts]: Thank you. Fixed.

File: tests/classes.spec.ts (hunk 2 above)
Comment (doz13189) [reply tests/classes.spec.ts]: Thank you. Fixed.

**Comment by doz13189** [comment]
@cexbrayat Thank you for the code review. I pushed the commit.

**Comment by cexbrayat** [comment]
@doz13189 Thanks 👍

Would you mind squashing these 3 commits into just one with a commit message like `test: add a test case for errorWrapper`?

(it avoids cluttering the history with several commits)

Once that done, I'll merge the PR

**Comment by doz13189** [comment]
@cexbrayat  I was confused by rebase. I can't do it well, so I'm going to make a new PR.

**Comment by doz13189** [comment]
Moved to https://github.com/vuejs/vue-test-utils-next/pull/424.

**Pull request closed without merging**